src/index.ts

```diff
@@ -1,4 +1,9 @@
-import { Client } from './client';
+import { Client, registerCustomElementsClient } from './client';
```

Comment:
I am registering custom elements from both the Client and the Host. Is there a better way to do this?